Gemfile

```diff
@@ -3,28 +3,18 @@ source "https://rubygems.org"
 gemspec
 
 group :development do
-
-  gem "bundler"
   gem "bump"
 
-  #gem "json", ">= 1.9"
   gem "pry-byebug"
 
-  gem "rspec", ">= 3"
+  gem "rspec"
   gem "nokogiri" # used to test html output
 
-  if RUBY_VERSION >= "2.5"
-    gem "sqlite3", "~> 1.4", platform: :mri
-  else
-    gem "sqlite3", "~> 1.3.6"
-  end
+  gem "sqlite3", ">= 1.6.0.rc2", platform: :mri
```

Comment:
Any reason this has to be `rc` version?

Reply:
Agree having `rc` sucks here, I chose it because sqlite3 **1.6.0.rc2** supports Ruby 3.2 on Mac:
https://rubygems.org/gems/sqlite3/versions/1.6.0.rc2-x86_64-linux

Last non-rc release — sqlite3 **1.5.4** — requires Ruby < 3.2 on Mac:
https://rubygems.org/gems/sqlite3/versions/1.5.4-x86_64-darwin